source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -274,6 +275,11 @@ def get_places(self, obj):
             if not type_ or type_ not in map_serializer:
                 logging.getLogger(__name__).error('Place not serialized (unknown type): {}'.format(feature))
                 continue
+            zone_type = feature.get('properties', {}).get('geocoding', {}).get('zone_type')
+            # TODO: do something smart with other zone type
+            if type_ == 'zone' and zone_type != 'city':
```

Comment:
you're sure you don't want the other ones ?

there are only cities in navitia for the moment ?

Reply:
I don't think we should return other type without returning the zone type in the response. 
Previously in mimir we didn't import something else than cities, so we keep the same behavior